INSTALL.md

```diff
@@ -1,82 +1,393 @@
 
+### Setting up msys2
+Open msys2 at C:\devkitPro\msys2\msys2_shell.bat.
 
-## Parallel builds
+Note that in msys2, Copy is Ctrl+Insert and Paste is Shift+Insert.
 
-See [the GNU docs](https://www.gnu.org/software/make/manual/html_node/Parallel.html) and [this Stack Exchange thread](https://unix.stackexchange.com/questions/208568) for more information.
+Certain packages are required to build pokeemerald. Install these by running the following command:
+```bash
+pacman -S make gcc zlib-devel git
+```
+(This command will ask for confirmation, just enter the yes action when prompted).
 
-To speed up building, run:
+Download [libpng](https://sourceforge.net/projects/libpng/files/libpng16/1.6.37/libpng-1.6.37.tar.xz/download).
 
-	make -j$(nproc)
+Change directory to where libpng was downloaded. By default, msys2 will start in the current user's profile folder, located at **C:\Users\\_<user>_**, where *\<user>* is your Windows username. In most cases, libpng should be saved within a subfolder of the profile folder. For example, if libpng was saved to **C:\Users\\_\<user>_\Downloads** (the Downloads location for most users), enter this command:
+```bash
+cd Downloads
+```
+(While not shown, msys uses forward slashes `/` instead of backwards slashes `\` as the directory separator. Windows path names are case-insensitive so adhereing to capitalization isn’t needed.  If the path has spaces, then the path must be wrapped with quotations, e.g. `cd "Downloads/My Downloads"`. If libpng was saved elsewhere, you will need to specify the full path to where libpng was downloaded, e.g. `cd c:/devkitpro/msys2` if it was saved there)
 
-`nproc` is not available on macOS. The alternative is `sysctl -n hw.ncpu` ([relevant Stack Overflow thread](https://stackoverflow.com/questions/1715580)).
+Run the following command to uncompress and install libpng.
+```bash
+tar xf libpng-1.6.37
+cd libpng-1.6.37
+./configure --prefix=/usr
+make check
+make install
+```
+Then finally, run the following command to change back to the user profile folder.
+```bash
+cd
+```
 
+### Choosing where to store pokeemerald (msys2)
+At this point, you can choose a folder to store pokeemerald into. If so, you'll have to change the **current working directory** every time you open msys2. If you're okay with storing pokeemerald in the user profile folder, then proceed to [Installation](#installation).
 
-## Debug info
+For example, if you want to store pokeemerald (and agbcc) in **C:\Users\\_\<user>_\Desktop\decomps**, enter this command:
+```bash
+cd Desktop/decomps
+```
+Note that the directory **must exist** in Windows. If you want to store pokeemerald in a dedicated folder that doesn't exist (e.g. the example provided above), then create the folder (e.g. using Windows Explorer) before executing the `cd` command.
 
-To build **pokeemerald.elf** with enhanced debug info:
+If this works, then proceed to [Installation](#Installation).
```

Comment:
All of the "proceed to Installation" links with a capital `I` in `Installation` don't appear to link correctly (Those with a lowercase `i` appear to work correctly)